packages/privacy-toolset/src/cookie-banner/styles.scss

```diff
@@ -0,0 +1,256 @@
+// Granular consent banner (v2).
+$granular-banner-max-width: 1024px;
+
+// Colors.
+$cookie-banner-bg-color: #FFFFFF;
+$cookie-banner-text-color: #000000;
+$cookie-banner-text-muted-color: #444444;
+$cookie-banner-link-color: #117ac9;
+
+// Buttons
+$cookie-banner-main-button-color: $cookie-banner-link-color;
+$cookie-banner-main-button-font-color: #FFFFFF;
+
+// Checkboxes
+$cookie-banner-checkbox-color: $cookie-banner-link-color;
+$cookie-banner-checkbox-bg-color: rgba(196, 195, 195, 0.55);
+
+// Fonts
+$cookie-banner-font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell",
+"Helvetica Neue", sans-serif;
+
+/**
+ Cookie Banner styling.
+*/
+.cookie-banner {
+	@keyframes fadeIn {
+		0% { opacity: 0; }
+		100% { opacity: 1; }
+	}
+
+	font-family: $cookie-banner-font-family;
+	display: flex;
+	position: fixed;
+	top: auto;
+	left: 50%;
+	transform: translate(-50%);
+	max-width: $granular-banner-max-width;
+	width:100%;
+	margin: 25px;
+	bottom: 25px;
+	margin: 0;
```

Comment:
Seems as though the margin is being set twice, once at 25px 2 lines up, and here at 0

Reply:
I think that's my bad, I'll fix it on the WPcom differential as well. 👍

Reply:
I'll clean this up after merging the newest changes from [WPcom diff](D85697-code).